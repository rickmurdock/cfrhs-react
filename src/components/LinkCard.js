import React from 'react';

const LinkCard = (props) => {
  return (
    <div className="card">
      <div className="card-header" role="tab" id={"heading" + props.card}>
        <h5 className="mb-0">
          <a data-toggle="collapse" data-parent="#accordion" href={"#collapse" + props.card} aria-expanded={props.expanded} aria-controls={"collapse" + props.card}>
            {props.title}
          </a>
        </h5>
      </div>
      <div id={"collapse" + props.card} className={"collapse " + props.show} role="tabpanel" aria-labelledby={"heading" + props.card}>
        <div className="card-block">
          <ul>
            {props.children}
          </ul>
        </div>
      </div>
    </div>
  );
};

export default LinkCard;