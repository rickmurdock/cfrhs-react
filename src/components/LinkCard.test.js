import React from 'react';
import renderer from 'react-test-renderer';
import LinkCard from './LinkCard';

test('LinkCard renders correctly', () => {
    const tree = renderer.create(
        <LinkCard />
    ).toJSON();
    expect(tree).toMatchSnapshot();
});