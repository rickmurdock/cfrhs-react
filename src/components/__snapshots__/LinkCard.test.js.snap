// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LinkCard renders correctly 1`] = `
<div
  className="card"
>
  <div
    className="card-header"
    id="headingundefined"
    role="tab"
  >
    <h5
      className="mb-0"
    >
      <a
        aria-controls="collapseundefined"
        aria-expanded={undefined}
        data-parent="#accordion"
        data-toggle="collapse"
        href="#collapseundefined"
      />
    </h5>
  </div>
  <div
    aria-labelledby="headingundefined"
    className="collapse undefined"
    id="collapseundefined"
    role="tabpanel"
  >
    <div
      className="card-block"
    >
      <ul
        className="ul-links"
      />
    </div>
  </div>
</div>
`;
